packages/react/src/SubNav/SubNav.test.tsx

```diff
@@ -200,4 +203,21 @@ describe('SubNav', () => {
 
     expect(queryByRole('link', {name: 'Copilot feature page one'})).not.toBeInTheDocument()
   })
+
+  it('renders an optional subheading into the document', () => {
+    const expectedText = 'Subheading'
+    const expectedLink = '#subheading'
+    const {getByText} = render(
+      <SubNav>
+        <SubNav.Heading href={headingLink}>{heading}</SubNav.Heading>
+        <SubNav.SubHeading href={expectedLink}>{expectedText}</SubNav.SubHeading>
+        <SubNav.Link href="#">Link</SubNav.Link>
+      </SubNav>,
+    )
+
+    const el = getByText(expectedText)
+
+    expect(el).toBeInTheDocument()
+    expect(el).toHaveAttribute('href', expectedLink)
+  })
 })
```

Comment:
If you get chance, it would be great to have a test which checks that focus is returned to the button on narrow submenu close too

Reply:
👍 Added